### DISCUSSION TITLE
FluxStore Multivendor - Add notice when store is closed

### DISCUSSION THREAD

**Discussion by Benjy04** [inspireui/support#5135] [Working on Features]
It would be nice if there would be an information on the product page and maybe also in the cart, when a store which sells the product is closed at the moment (wcfm opening hours)

**Comment by pideloahora** [comment]
Perfect

**Comment by Haniel5** [comment]
Hi, the closed/opened vendor feature is available on FluxStore Multi-Vendor:  https://1.envato.market/NW4q1 - Thank you!
